### DISCUSSION TITLE
Ability to go back to previously selected beatmap after hitting F2(random select).

### DISCUSSION THREAD

**Discussion by ArchUsr64** [ppy/osu#14760] [Ideas] [answered]
A pretty minor feature suggestion but as far as I know there is no way to go back to the previously selected beatmap after doing a random select. 
A way to go back to the previously selected beatmap would be a nice QoL addition, possible keybindings could be `shift+F2` or right clicking the "random" button.

**Comment by peppy** [comment, marked as answer]
This already exists, and is already set by default to Shift+F2. Please check your key bindings.
